roles/swift-account/tasks/main.yml

```diff
@@ -38,4 +39,14 @@
     - swift-account-auditor
     - swift-account-reaper
     - swift-account
-  when: start_account|bool
+  when: start_account|bool and ansible_os_family == 'Debian'
+
+
+- name: start swift-account services
```

Comment:
follow oher roles, where it will need only one task to deal with start services based on os.